pure.zsh

```diff
@@ -701,7 +701,8 @@ prompt_pure_state_setup() {
 
 # Return true if executing inside a Docker or LXC container.
 prompt_pure_is_inside_container() {
-	([[ -r /proc/1/cgroup ]] && grep -q -E "(lxc|docker)" /proc/1/cgroup ) \
+	local -r cgroup_file='/proc/1/cgroup'
+	[[ -r "$cgroup_file" && -n "${(M)$(< $cgroup_file):#*(lxc|docker)*}" ]] \
```

Comment:
This is a good idea. I think this could be made a bit easier to read:

```suggestion
	[[ -r "$cgroup_file" && [[ "$(<$cgroup_file)" = *(lxc|docker)* ]] \
```

Although I can appreciate that the original solution is quite elegant too, but requires deeper knowledge of zsh syntax 😄.

Reply:
Thanks to your review! And your suggestion is certainly true. It is very important to read easily. I will fix it 😄.

By the way, is it typo that the second double open brackets on the above suggested change? I think it will cause a syntax error...🤔

Reply:
Good catch, it was a typo in the sense that I meant to close the previous one (`]] && [[`) 😄. Combining like you did is also fine 👍🏻.

Reply:
OK!